tests/org/mozilla/io/TestLocalMsgProtocol.java

```diff
@@ -405,24 +335,11 @@ public void test(TestHarness th) {
 
             // Scenario 3
             clientThread.start();
-            synchronized (openLock) {
-                while (!clientCreated) {
-                    openLock.wait();
-                }
-            }
             serverCreateThread.start();
-            synchronized (openLock) {
-                while (!serverCreated) {
-                    openLock.wait();
-                }
-            }
-            serverAcceptAndOpenThread.start();
             serverCreateThread.join();
+            serverAcceptAndOpenThread.start();
             serverAcceptAndOpenThread.join();
             clientThread.join();
```

Comment:
Scenario 3 was:
1) Client opens a localmsg server
2) Server creates a localmsg server
3) Server calls acceptAndOpen

With this change, 2 could happen before 1 and 3 could happen before 1.